pulsar-client/src/main/java/org/apache/pulsar/client/impl/MemoryLimitController.java

```diff
@@ -68,6 +72,15 @@ public boolean tryReserveMemory(long size) {
         }
     }
 
+    private static void ensureReservedMemoryIsPositive(long reservedMemorySize) {
+        if (reservedMemorySize < 0) {
```

Comment:
Nit： Zero is not positive but can pass this method. So change to `reservedMemorySize <= 0` here or change the method name:)

Reply:
Good suggestion, skipped all logic if the argument is `0`